src/components/NavBar/ChainSelector.tsx

```diff
@@ -13,24 +14,38 @@ import { useOnClickOutside } from 'hooks/useOnClickOutside'
 import useSelectChain from 'hooks/useSelectChain'
 import useSyncChainQuery from 'hooks/useSyncChainQuery'
 import { useAtomValue } from 'jotai/utils'
-import { Box } from 'nft/components/Box'
 import { Portal } from 'nft/components/common/Portal'
-import { Column, Row } from 'nft/components/Flex'
+import { Column } from 'nft/components/Flex'
 import { useIsMobile } from 'nft/hooks'
 import { useCallback, useMemo, useRef, useState } from 'react'
 import { AlertTriangle, ChevronDown, ChevronUp } from 'react-feather'
-import { useTheme } from 'styled-components'
+import styled, { useTheme } from 'styled-components'
 import { getSupportedChainIdsFromWalletConnectSession } from 'utils/getSupportedChainIdsFromWalletConnectSession'
 
-import * as styles from './ChainSelector.css'
 import ChainSelectorRow from './ChainSelectorRow'
 import { NavDropdown } from './NavDropdown'
 
 const NETWORK_SELECTOR_CHAINS = [...L1_CHAIN_IDS, ...L2_CHAIN_IDS]
 
-interface ChainSelectorProps {
-  leftAlign?: boolean
-}
+const ChainSelectorWrapper = styled.div`
+  position: relative;
+`
+
+const ChainSelectorButton = styled(BaseButton)<{ isOpen: boolean }>`
+  display: flex;
+  background: ${({ theme, isOpen }) => (isOpen ? theme.accent2 : `none`)};
```

Comment:
nit:
```suggestion
  background: ${({ theme, isOpen }) => (isOpen ? theme.accent2 : 'none')};
```

Reply:
Also this background color when opened is new, making sure it's intentional?

Reply:
i was also surprised to find that it was in the figma: https://www.figma.com/file/4DJrdsx1lPm5u409xyKDZj/%F0%9F%8D%84-%F0%9F%95%B8%EF%B8%8F-Spore---Web?type=design&node-id=1784%3A36695&mode=design&t=ZjnH9nOnA8LUzT8d-1

also it was part of the old code but wasn't displaying correctly